ios/FluentUI/Notification/MSFNotificationView.swift

```diff
@@ -0,0 +1,194 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+import UIKit
+
+/// UIKit wrapper that exposes the SwiftUI `Notification` implementation
+@objc open class MSFNotification: NSObject, FluentUIWindowProvider {
+
+    /// The UIView representing the Notification.
+    @objc open var view: UIView {
+        return hostingController.view
+    }
+
+    /// Creates a new MSFNotification instance.
+    /// - Parameters:
+    ///   - style: The MSFNotification value used by the Notification.
+    ///   - message: The primary text to display in the Notification.
+    ///   - theme: The FluentUIStyle instance representing the theme to be overriden for this Notification.
+    @objc public init(style: MSFNotificationStyle,
+                      message: String,
+                      theme: FluentUIStyle? = nil) {
+        super.init()
+
+        notification = NotificationViewSwiftUI(style: style, message: message, dismissAction: { self.hide() })
+        hostingController = FluentUIHostingController(rootView: AnyView(notification
+                                                                            .windowProvider(self)
+                                                                            .modifyIf(theme != nil, { notification in
+                                                                                notification.customTheme(theme!)
+                                                                            })))
+        hostingController.view.backgroundColor = UIColor.clear
+        hostingController.disableSafeAreaInsets()
+    }
+
+    public var state: MSFNotificationState {
+        get {
+            return notification.state
+        }
+
+        set {
+            notification.state = newValue as! MSFNotificationStateImpl
+        }
+    }
+
+    public var style: MSFNotificationStyle {
+        get {
+            return notification.state.style
+        }
+
+        set {
+            notification.state.style = newValue
+            notification.tokens.style = newValue
+        }
+    }
+
+    public var delayTime: TimeInterval = 2
+
+    // MARK: - FluentUIWindowProvider
+
+    var window: UIWindow? {
+        return self.view.window
+    }
+
+    // MARK: - Show/Hide Methods
+
+    public func showNotification(in view: UIView, completion: ((MSFNotification) -> Void)? = nil) {
+        if isShown {
```

Comment:
Use guard for early return.
https://github.com/microsoft/swift-guide/blob/gh-pages/EarlyReturn.md